src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -6337,6 +6338,100 @@ void AutoCalcDOASControlStrategy(EnergyPlusData &state)
     }
 }
 
+void ReportInfiltrations(EnergyPlusData &state)
```

Comment:
This function is created to update object-level reporting data, it follows the same equations used in zone reporting.